src/Core/Framework/Adapter/Cache/Http/CacheResponseSubscriber.php

```diff
@@ -223,36 +223,26 @@ private function hasInvalidationState(array $cacheStates, array $states): bool
 
     private function buildCacheHash(Request $request, SalesChannelContext $context): string
     {
-        if (Feature::isActive('cache_rework')) {
```

Comment:
is it okay to directly use it without the feature activated? 🤔

Reply:
Yes, right now the outcome of the hash is the "public api" (which is not really extendable). So right now just the values are considered to be a break when they would change - but this is not the case here. So here i just activated the new config and the new event